install/kubernetes/hubble/values.yaml

```diff
@@ -67,9 +67,10 @@ ui:
   #         cpu: 100m
   #         memory: 64Mi
   resources: {}
+  
   # The priority class system-node-critical marks add-on pods as critical to the node itself.
   # This priority class is only valid under the kube-system namespace.
-  priorityClassName: ""
+  priorityClassName: system-node-critical
```

Comment:
This priority class is only available in the `kube-system` namespace so with the namespace name check removed, I think this is going to fail.

Reply:
indeed, my bad this was not intended 🤦